src/views/33Together/PoolWithdraw.jsx

```diff
@@ -144,30 +146,32 @@ export const PoolWithdraw = props => {
               {/* Withdraw sOHM */}
             </Button>
           </Box>
+          {newOdds > 0 && quantity > 0 && (
+            <Box padding={1}>
+              <Typography color="error" variant="body2">
+                Withdrawing {quantity} sOHM reduces your odds of winning to 1 in {newOdds}&nbsp;
+              </Typography>
+            </Box>
+          )}
           {exitFee > 0 && (
-            <Box margin={2}>
+            <Box margin={1}>
               <Typography color="error">
-                Oh no! You're attempting to withdraw early and therefore, are subject to a fairness fee of {exitFee}.
-                &nbsp;
+                Early withdraw will incur a fairness fee of {exitFee}. &nbsp;
                 <Link
                   href="https://docs.pooltogether.com/protocol/prize-pool/fairness"
                   target="_blank"
                   rel="noreferrer"
                   color="primary"
                 >
-                  Read more about Fairness
+                  <br />
+                  Read more about Fairness{" "}
```

Comment:
would you rather me start using `{" "}` rather than `&nbsp;`

Reply:
I dont have a preference really, i think the {" "} is slightly easier to read